### PR TITLE
docs(ProgressBar): fix title

### PR DESCRIPTION
**Pull request by layershifter**
## Previous Behavior

image

## New Behavior

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Avatar-mount_base.html">642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Button-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Button-mount.html">295</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Field-mount_base.html">1105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Field-mount.html">1166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProvider-mount_base.html">684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProvider-mount.html">712</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-mount_base.html">80</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-mount.html">81</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">59</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">63</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">76</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">75</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/MakeStyles-mount_base.html">895</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/MakeStyles-mount.html">877</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Persona-mount_base.html">1713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/Persona-mount.html">1726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/SpinButton-mount_base.html">1378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30658/perf-test-react-components/SpinButton-mount.html">1401</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `70.732 kB`<br />`20.465 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `217.207 kB`<br />`61.943 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `43.219 kB`<br />`14.299 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `7.944 kB`<br />`2.588 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/ProgressBar.fixture.js'>ProgressBar</abbr> | `17.428 kB`<br />`6.898 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4596285379d224fb7ed27b459db0ee1ed90160fa'>4596285379d224fb7ed27b459db0ee1ed90160fa</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-progress/stories/ProgressBar/ProgressBarBestPractices.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <details>
```
Comment (fabricteam) [inline packages/react-components/react-progress/stories/ProgressBar/ProgressBarBestPractices.md:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30658&commitId=ad13930ff5d6391f1061b31ec8e8163ee53ded5f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+image+badge.normal.chromium.png|725| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by khmakoto — APPROVED** [review]

File: packages/react-components/react-progress/stories/ProgressBar/ProgressBarBestPractices.md (hunk 1 above)
Comment (layershifter) [reply packages/react-components/react-progress/stories/ProgressBar/ProgressBarBestPractices.md:1]: clearly not related

**Pull request merged**